vast.go

```diff
@@ -96,6 +102,30 @@ func (v *VAST) RemovePixels() {
 	}
 }
 
+func (v *VAST) GetCreativeType() string {
```

Comment:
This method should return an error instead of an empty string when there's no match.

Reply:
Fixed